server/modules/user/api/userApi.js

```diff
@@ -72,6 +72,21 @@ module.exports.init = app => {
     }
   })
 
+  app.get('/user/:userUuid/profilePicture', async (req, res, next) => {
+    try {
+      const { userUuid } = Request.getParams(req)
+
+      const { profile_picture: profilePicture } = await UserService.fetchUserProfilePicture(userUuid)
```

Comment:
we use the convention that we return objects whose names are camelCased. but why ```UserService.fetchUserProfilePicture``` returns an object with a property? wouldn't it better to just return the picture?

Reply:
Right, returning the picture

Reply:
what i meant is that ```UserService.fetchUserProfilePicture``` returns the file itself, not an object with a property called ```profile_picture```. this would become:
```const profilePicture = await UserService.fetchUserProfilePicture(userUuid)```

Reply:
Ok, for some reason I had stashed the change